openshift-fluentd-syslog-journal.conf

```diff
@@ -0,0 +1,25 @@
+<source>
+  @type systemd
+  path "#{ENV['DATA_DIR']}/journal"
+  pos_file "#{ENV['DATA_DIR']}/journal.pos"
+  tag system.var.log.messages
```

Comment:
So that the tag will exactly resemble the case where fluentd is reading from /var/log/messages*, for the other filters and outputs.